pvr.vuplus/resources/language/resource.language.en_us/strings.po

```diff
@@ -131,3 +131,35 @@ msgstr "Disconnected from '%s'"
 msgctxt "#30501"
 msgid "Reconnected to '%s'"
 msgstr "Reconnected to '%s'"
+
+msgctxt "#30050"
+msgid "Custom Live TV timeout (0 to use default)"
+msgstr ""
+
+msgctxt "#30060"
+msgid "Timeshift"
+msgstr "Timeshift"
+
+msgctxt "#30061"
+msgid "Enable Timeshift"
+msgstr "Enable Timeshift"
+
+msgctxt "#30062"
+msgid "Timeshift buffer path"
+msgstr "Timeshift buffer path"
+
+msgctxt "#30063"
+msgid "Off"
+msgstr "Off"
+
+msgctxt "#30064"
+msgid "On playback"
+msgstr "On playback"
+
+msgctxt "#30065"
+msgid "On pause"
+msgstr "On pause"
+
+msgctxt "#30514"
+msgid "Timeshift buffer path does not exist"
+msgstr ""
```

Comment:
Please revert the changes to this file. The source language for this project is en_GB and only that strings.po needs to modified by the deveopers. All other languages are provided via Transifex and those changes would overwrite what you have changed here.